src/udev/cdrom_id/cdrom_id.c

```diff
@@ -687,26 +698,23 @@ static int cd_profiles(int fd) {
         unsigned cur_profile = 0;
         unsigned len;
         unsigned i;
-        int err;
-        int ret;
-
-        ret = -1;
+        int r, ret = -1;
```

Comment:
nit: maybe change ```ret``` to a ```bool have_media```? It's much nicer for readers to avoid having 2 return ints

Reply:
nvm I see you did exactly this in a later commit :-) going through the PR commit-by-commit